### PR TITLE
Release v3.0.0

### PR DESCRIPTION
**Pull request by charlierudolph**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
 
 ## Unreleased
 
+## 3.0.0 (2016-02-07)
+
+#### BREAKING CHANGES
+* `git hack`: no longer accepts a parent branch
```
Comment (allewun) [inline RELEASE_NOTES.md]: can you also include a reference to the issue so users can see the discussion and rationale for the change?

Comment (charlierudolph) [inline RELEASE_NOTES.md]: I looked through the issues before making this PR and there wasn't any discussion for this in particular. There is an issue for [git prepend](https://github.com/Originate/git-town/issues/766) but `git append` and the change to `git hack` I think just came out of the PRs. @kevgo were you working off any particular issues for those changes?

Comment (charlierudolph) [inline RELEASE_NOTES.md]: I guess this [comment](https://github.com/Originate/git-town/pull/778#issuecomment-269514502) is the where it started

Comment (charlierudolph) [inline RELEASE_NOTES.md]: Thoughts on just adding updating this to:
```
 `git hack`: no longer accepts a parent branch (use `git append` instead)
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -2,6 +2,20 @@
 
 ## Unreleased
 
+## 3.0.0 (2016-02-07)
```
Comment (allewun) [inline RELEASE_NOTES.md]: @charlierudolph didn't catch it earlier :( but wrong year here and in the manfiles

**Pull request closed without merging**